runbotics/runbotics-scheduler/src/scheduler-database/process-credential/process-credential.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { TenantInterceptor } from '#/utils/interceptors/tenant.interceptor';
+import { Logger } from '#/utils/logger';
+import { Body, Controller, Delete, Get, Param, ParseIntPipe, ParseUUIDPipe, Post, UseInterceptors } from '@nestjs/common';
+import { ProcessCredentialService } from './process-credential.service';
+import { IProcess } from 'runbotics-common';
+import { User } from '#/utils/decorators/user.decorator';
+import { UserEntity } from '#/database/user/user.entity';
+import { ZodValidationPipe } from '#/utils/pipes/zod-validation.pipe';
+import { CreateProcessCredentialDto, createProcessCredentialSchema } from './dto/create-process-credential.dto';
+
+
+@UseInterceptors(TenantInterceptor)
+@Controller('api/scheduler/tenants/:tenantId/process-credentials')
+export class ProcessCredentialController {
+    private readonly logger = new Logger(ProcessCredentialController.name);
+    constructor(private readonly processCredentialService: ProcessCredentialService) {}
+
+    @Get('processes/:processId')
```

Comment:
this endpoint belongs to process controller: `processes/:processId/credentials`,  `@Get('processes/:processId')` suggests it returns a process